tests/explainers/test_kernel.py

```diff
@@ -7,6 +7,10 @@
 import shap
 
 
+def sigm(x):
+    return np.exp(x) / (1 + np.exp(x))  # noqa: E731
```

Comment:
Nitpick: I think the E731 ignore is not needed, as I don't see a lambda expression here

```suggestion
    return np.exp(x) / (1 + np.exp(x))
```